includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -675,13 +675,66 @@ public function is_connected() {
 	/**
 	 * Checks if the account has not completed onboarding due to users abandoning the process half way.
 	 * Also used by WC Core to complete the task "Set up WooPayments".
+	 * Called directly by WooCommerce Core.
 	 *
 	 * @return bool
 	 */
 	public function is_account_partially_onboarded(): bool {
 		return $this->account->is_stripe_connected() && ! $this->account->is_details_submitted();
 	}
 
+	/**
```

Comment:
I moved these from below to colocate them all close to each other. It's not necessary, but thought it would be nice to colocate them 🤷